apps/condo/domains/ticket/constants/errors.js

```diff
@@ -1,6 +1,4 @@
 
-const EMPTY_DATA_EXPORT_ERROR = '[export.empty.data'
```

Comment:
This is empty human-generated file. I don't think we should keep it

Reply:
done